app/models/review.rb

```diff
@@ -3,7 +3,21 @@ class Review < ActiveRecord::Base
   belongs_to  :movie
 
   validates   :title,
-              :content, presence: true
+              :content,
+              presence: true
 
   validates   :rating, numericality: { greater_than_or_equal_to: 0.0, less_than_or_equal_to: 10.0, precision: 1 }
+
+  before_save   :set_user
+  after_commit  :update_movie_rating
```

Comment:
`after_commit` is called after a record has been created, updated, or destroyed.